### ISSUE TITLE
New maintainer

### ISSUE THREAD

**Issue by Ghosuto** [Project-Awaken/official_devices#68]
## Device you want to apply for
```
Device :-  POCO F3 / Mi 11X
Device Code name :- Alioth / Aliothin
```

## Device tree sources

```
https://github.com/Ghosuto/device_xiaomi_alioth
https://github.com/Ghosuto/device_xiaomi_sm8250-common
https://github.com/Ghosuto/android_vendor_xiaomi_alioth
https://github.com/Ghosuto/android_vendor_xiaomi_sm8250-common
https://github.com/EmanuelCN/kernel_xiaomi_sm8250/tree/staging
```

## Link to the platform where you have released your unofficial build
```
https://t.me/pocoF3GlobalUpdates/1646
```

## Other ROM's you maintain officially or unofficially
```
None
```

## Bugs on unofficial build
```
None
```

## How long have you been building for?
```
6 Months
```

## Did you properly read [Device stability requirements](https://github.com/Project-Awaken/official_devices/blob/12/requirements.md) and [Maintainer requirements](https://github.com/Project-Awaken/official_devices/blob/12/maintainerreq.md) before applying? What part of those requirements do you violate?
```
I read everything properly and I violate none of the requirements.
```

## SELinux status?
```
Enforcing.
```

## Are you using a prebuilt kernel? If yes, why?
```
No.
```

### Telegram username
```
@Ghosuto7
```

**Comment by Ghosuto** [comment]
Kk tnx bye....me done 👍

**Comment by Ghosuto** [comment]
Kek tnx bye..me done 👍

**Comment by Ghosuto** [comment]
Tnx... I'm done....
 now...

**Comment by KrishnakantShedge** [comment]
1. Fix your commit format. It's inconsistent with alioth and Alioth.
https://github.com/Ghosuto/device_xiaomi_alioth/commit/74d3b1702fdd92d8a4cfe42eba9f5c7cc2cc4bf5
https://github.com/Ghosuto/device_xiaomi_sm8250-common/commit/52a5f573c210d1073aaa6ffacab44f24a87d94bb
https://github.com/Ghosuto/device_xiaomi_sm8250-common/commit/e530ab484c7712311befef4c2ee1b7ab2645a62a
2. We don't have quick tap in 13
https://github.com/Ghosuto/device_xiaomi_alioth/commit/6790551c292db1dd495610faacb71ec442175dc7
3. No prebuilt OEM apps please which includes dolby as well
https://github.com/Ghosuto/device_xiaomi_alioth/commit/1f60ee937d1721aee6a6244c24f8fec4293ccb54
4. Drop this:
https://github.com/Ghosuto/device_xiaomi_sm8250-common/commit/21a02052bab6ad91ecb30c89020a9099871b465b
https://github.com/Ghosuto/device_xiaomi_sm8250-common/commit/0c87ced7b2e943a5ac4a4ce28069eb49646a73ba
5. Give proper author:
https://github.com/Ghosuto/device_xiaomi_sm8250-common/commit/d42134262cccd09cb5b247a6fb9e06954b13e8da
6. Explain me if you have really pick these commits in the common tree

**Comment by Ghosuto** [comment]
Fixed all commits you mentioned, 
and for Dolby 

1. I didn't saw any rules about **No OEM apps ** in Awaken rules when I applied. (sorry if I missed)
2. Most of the alioth(PocoF3) roms comes with dolby inbuilt except Arrow and official PE as far as I know.
3. There is no issue in dolby. It's working perfectly.
4. Most of the users (more then 80%, source - PocoF3 global chat group) prefer dolby inbuild roms more normal one (no dolby) because of better sound quality and if I drop dolby most of the users gonna ignore awaken because of that. So for me,... I don't think drop dolby is a good idea.
